packages/plugin-forms-api/src/models/definitions/constants.ts

```diff
@@ -0,0 +1,4 @@
+export const TEXT_TYPE = [
```

Comment:
inappropriate variable name TEXT_TYPE, change this to INPUT_TYPE, CONTROL_TYPE or something